docs/technical-documentation/updating_drupal.md

```diff
@@ -0,0 +1,67 @@
+* Optionally, if you made manual modifications to files like .htaccess, composer.json, or robots.txt, copy them somewhere easy to find. Because after you've installed the new Drupal core, you will need to re-apply the changes. For example, Acquia Dev Desktop places a .htaccess file in the top-level directory and without it, only the homepage on your site will work.
+
+**Warning:** Always revert to a backup if you get a fatal error in the update process.
+
+## What is Composer
+It is recommended by Drupal.org and the Islandora 8 community to use Composer with Drupal 8 for various tasks. 
+
+"[Composer](https://getcomposer.org/) is a [dependency manager](https://en.wikipedia.org/wiki/Package_manager) for PHP. Drupal core uses Composer to manage core dependencies like Symfony components and Guzzle."
+
+Taken from: https://www.drupal.org/docs/develop/using-composer/using-composer-with-drupal
+
+## Updating Drupal Core
+Over time new versions of Drupal “core” are released, and Islandora users are encouraged to install official Drupal core updates and security patches. On the other hand “alpha” and “beta" versions of Drupal core should only be installed if by advanced users for testing purposes. 
+
+The Islandora community STRONGLY recommends that the "Composer" method of upgrading Drupal core be used with Islandora 8 as mentioned here:
+
+https://www.drupal.org/docs/8/update/update-core-via-composer
+
+### Here is an overview of the steps for updating Drupal core using Composer
+
+NOTE: First make sure you have made database and file back ups.
+
+1) First, verify that an update of Drupal core actually is available:
+
+`composer outdated "drupal/*"`
+
+If there is no line starting with drupal/core, Composer isn't aware of any update. If there is an update, continue with the commands below.
+
+If there is no line starting with drupal/core, Composer isn't aware of any update. If there is an update, continue with the commands below.
+
+2) Assuming you are used to updating Drupal and know all the precautions that you should take, the update is as simple as:
+
+`composer update drupal/core webflo/drupal-core-require-dev "symfony/*" --with-dependencies`
+
+If you want to know all packages that will be updated by the update command, use the --dry-run option first.
+
+NOTE: Islandora 8 is configured to use a fork of drupal-composer/drupal-project which requires a specific composer syntax used above compared to other Drupal 8 sites. In addition, if you are upgrading from 8.5 to 8.7, you need to replace "~8.5.x" with "^8.7.0" for drupal/core and webflo/drupal-core-require-dev in composer.json. 
+
+Taken from:
+https://www.drupal.org/docs/8/update/update-core-via-composer#s-one-step-update-instruction
+
+3) Apply any required database updates using drush updatedb, or use the web admin user interface.  
+
+`drush updatedb`
+
+4) Clear the cache using drush cache:rebuild, or use the web admin user interface.
```

Comment:
putting `` marks around snippets can make them a little easier to pick out from the text, like: ` drush cache:rebuild`